css/chat.css

```diff
@@ -1,761 +1,780 @@
-    outline-offset: 0;
+  display: none;
+  background: transparent;
+  color: #666;
+  border-radius: 25px;
+  margin: 30px 0 0;
+  outline: none;
+  outline-offset: 0;
 }
 
 .show-load-more .msg-chats .load-more {
-    display: inline-block;
+  display: inline-block;
 }
 
 .msg-chats .load-more:active:focus,
 .msg-chats .load-more:focus {
-    outline: none;
-    outline-offset: 0;
+  outline: none;
+  outline-offset: 0;
 }
 
 .chats .chat.chat-divider {
-    justify-content: center;
-    color: #333;
-    opacity: 0.35;
+  justify-content: center;
+  color: #333;
+  opacity: 0.35;
 }
 
-.chats .chat + .chat.chat-divider {
-    margin-top: 20px;
+.chats .chat+.chat.chat-divider {
+  margin-top: 20px;
 }
 
-.chat.chat-divider + .chat {
-    margin-top: 20px;
+.chat.chat-divider+.chat {
+  margin-top: 20px;
 }
 
 .chats .chat.chat-divider .timestamp {
-    font-size: 0.8em;
-    padding: 2px 6px;
-    background-color: rgba(51, 51, 51, 0.16);
-    border-radius: 25px;
+  font-size: 0.8em;
+  padding: 2px 6px;
+  background-color: rgba(51, 51, 51, 0.16);
+  border-radius: 25px;
 }
+
 @media screen and (min-width: 640px) {
-    body {
-        padding-top: 55px;
-    }
-
-    .convo-aside {
-        width: 33.333%;
-        border-right: 1px solid #e4eaec;
-    }
-
-    .msg-area {
-        position: relative;
-        width: 66.667%;
-        flex: 1;
-        left: 0;
-        z-index: 9;
-    }
-
-    .overlay-open .convo-aside {
-        margin-left: 0;
-    }
-
-    .overlay-open .convo-aside .screen {
-        opacity: 0;
-        visibility: hidden;
-    }
-
-    .chat .chat-body {
-        max-width: 80%;
-    }
-
-    .list-group li.active {
-        padding: 15px 13px 15px 82px;
-        border-right: 2px solid #337ab7;
-        background-color: rgba(51, 122, 183, 0.06);
-    }
-
-    .list-group li .title-holder .user-name {
-        font-size: 1.1em;
-        margin: 0 15px 8px 0;
-    }
-
-    .list-group li .convo-preview p {
-        font-size: 1em;
-    }
-
-    .fl-with-top-menu .msg-area .profile-header {
-        top: 55px;
-    }
-
-    .msg-area .profile-header .back-btn {
-        display: none;
-    }
+  body {
+    padding-top: 44px;
+  }
+
+  .convo-aside {
+    width: 33.333%;
+    border-right: 1px solid #e4eaec;
+  }
+
+  .msg-area {
+    position: relative;
+    width: 66.667%;
+    flex: 1;
+    left: 0;
+    z-index: 9;
+  }
+
+  .overlay-open .convo-aside {
+    margin-left: 0;
+  }
+
+  .overlay-open .convo-aside .screen {
+    opacity: 0;
+    visibility: hidden;
+  }
+
+  .chat .chat-body {
+    max-width: 80%;
+  }
+
+  .list-group li.active {
+    padding: 15px 13px 15px 82px;
+    border-right: 2px solid #337ab7;
+    background-color: rgba(51, 122, 183, 0.06);
+  }
+
+  .list-group li .title-holder .user-name {
+    font-size: 1.1em;
+    margin: 0 15px 8px 0;
+  }
+
+  .list-group li .convo-preview p {
+    font-size: 1em;
+  }
+
+  .fl-with-top-menu .msg-area .profile-header {
+    top: 44px;
```

Comment:
As above. This may be unnecessary.

Reply:
@tonytlwu That one is needed. On mobile is `0` because it opens in an overlay, on tablet is the size of the top bar because it opens on the side.